website/docs/r/iam_access_group_account_settings.html.markdown

```diff
@@ -0,0 +1,33 @@
+---
+subcategory: "IAM Access Groups"
```

Comment:
I think we have this as subcategory
```
Identity & Access Management (IAM)
```
Can you update it

Reply:
@hkantare updated the file